collect_numbers.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+import argparse
+import os
+import shlex
+import subprocess as sp
+import sys
+
+def handle_exception(error):
+    print ("[AggregatorError] Output : ", error.stdout)
+    print ("[AggregatorError] Error : ", error.stderr)
+
+
+def invoke_iree_compile(my_env, compile_script, compile_args = [],
+                        verbose = False):
+    args = [compile_script, 'gfx942'] + compile_args
+    if verbose:
+        print(f"[Aggregator] Running : {' '.join(args)} ")
```

Comment:
joining will make it hard to tell what the arguments are because it will break the separation between list elements and won't add any quotes to help -- maybe just print it as a list?

Reply:
maybe, but this is a bit easier to read. Ill leave as is...